OpenRA.Game/Map/TileSet.cs

```diff
@@ -84,7 +84,7 @@ public class TerrainTemplateInfo
 		public readonly int[] Frames;
 		public readonly int2 Size;
 		public readonly bool PickAny;
-		public readonly string Category;
+		public readonly string[] Category;
```

Comment:
Please rename this to `Categories` and an entry to `UpgradeRules.UpgradeTileset` to convert the yaml.